src/Graduacao.php

```diff
@@ -503,4 +503,30 @@ public static function listarDisciplinasGradeCurricular($codcur, $codhab, $tipob
         ];
         return DB::fetchAll($query, $param);
     }
+
+
+    /**
+     * Retorna lista com os intercâmbios internacionais ativos de alunos da Graduação, 
+     * com número USP do aluno, nome da Universidade e nome do país da Universidade
+     *
+     * @param Int $codundclgi: código da unidade
+     * @return Array
+     * @author @gabrielareisg 28/05/2021
+     **/
+    public static function listarIntercambios($codundclgi)
```

Comment:
@gabrielareisg você pegar $codundclgi da variável de ambiente e usar IN.
Veja outros métodos que usam $codundclgi 

Outra coisa, pode deixar a query na pasta resources, o que acha?